Gopkg.toml

```diff
@@ -93,3 +93,7 @@
 [[constraint]]
   name = "k8s.io/ingress"
   revision = "0c6f15e372c831de52fcc393932540bb3a6d51b5"
+
+[[constraint]]
+  name = "github.com/russross/blackfriday/"
```

Comment:
why do we need that ? (markdown processor)

Reply:
can you please not mix-up these  godep stuff with the actual PR?

Reply:
I cannot not mix it, because I need to build. Why does this bother you?